stdlib/2/sys.pyi

```diff
@@ -6,6 +6,10 @@ from typing import (
 )
 from types import FrameType, ModuleType, TracebackType, ClassType
 
+# The following type alias are stub-only and do not exist during runtime
+ExcInfo = Tuple[Type[BaseException], BaseException, TracebackType]
```

Comment:
Then we should prefix these with an underscore.

Reply:
I have a purity vs practicability question. Reading PEP 3333 leads me to believe that `StartResponse` should use `_ExcJnfo`, not `_OptExcInfo` like it currently does. But that would mean that it can't be easily called using `sr(..., ..., sys.exc_info())` from an error handler. On the other hand, this way the responsibility is shifted to the server to check in the start response implementation that both exc_info as well as the tuple items are not `None` . But then that's probably a good idea anyway?